core/src/test/java/org/apache/druid/data/input/impl/LocalInputSourceTest.java

```diff
@@ -98,11 +99,19 @@ public void testGetFileIteratorWithBothBaseDirAndDuplicateFilesIteratingFilesOnl
     File baseDir = temporaryFolder.newFolder();
     List<File> filesInBaseDir = new ArrayList<>();
     for (int i = 0; i < 10; i++) {
-      filesInBaseDir.add(File.createTempFile("local-input-source", ".data", baseDir));
+      final File file = File.createTempFile("local-input-source", ".data", baseDir);
+      try (FileWriter writer = new FileWriter(file)) {
```

Comment:
The forbidden apis checks is flagging this: `java.io.FileWriter [Uses default charset]`

Reply:
Fixed.